cmd/run_konan

```diff
@@ -89,13 +89,13 @@ SHARED_JAR="${KONAN_HOME}/konan/lib/shared.jar"
 EXTRACTED_METADATA_JAR="${KONAN_HOME}/konan/lib/konan.metadata.jar"
 EXTRACTED_SERIALIZER_JAR="${KONAN_HOME}/konan/lib/konan.serializer.jar"
 KLIB_JAR="${KONAN_HOME}/konan/lib/klib.jar"
-KLIOPT_JAR="${KONAN_HOME}/konan/lib/kliopt-jvm.jar"
+KOTLINX_CLI_JAR="${KONAN_HOME}/konan/lib/kotlinx.cli-jvm.jar"
```

Comment:
Could we eventually repack  all them into single jar?

Reply:
I think it should be done a bit later, when we start to change mechanism of using this library.